packages/golang/build.yaml

```diff
@@ -6,7 +6,7 @@ requires:
 prelude:
 {{ if .Values.tool_image_distribution }}
 {{if eq .Values.tool_image_distribution "opensuse" }}
-- zypper in -y curl git && zypper install -y -t pattern devel_basis
+- zypper in -y curl git gzip && zypper install -y -t pattern devel_basis
```

Comment:
Not sure what happened here, but apparently we need to include `gzip` now... I guess this is another unexpected change on teal base image 😓

Reply:
Is that also needed in `builder-image` ? 🤔